languages/PYTHON.md

```diff
@@ -865,4 +865,14 @@ Example:
 
 [![Saleor](http://getsaleor.com/mr-saleor-readme.png)](http://getsaleor.com)
 
```

Comment:
Separator needed here?

Reply:
No, there is no need. It looks nice. Thank you!

Reply:
Yeah, actually is needed.

Reply:
Fixed in #90 , whoops!